src/features/user/BulkCodes/forms/IssueCodesForm.tsx

```diff
@@ -52,7 +60,11 @@ const IssueCodesForm = (): ReactElement | null => {
   const [isSubmitted, setIsSubmitted] = useState(false);
   const [isEditingRecipient, setIsEditingRecipient] = useState(false);
   const [isAddingRecipient, setIsAddingRecipient] = useState(false);
+  const [notificationLang, setNotificationLang] = React.useState('en');
 
+  const handleLocale = (event: SelectChangeEvent) => {
+    setNotificationLang(event.target.value as string);
+  };
```

Comment:
1. The name could be more appropriate. `handleLocale` sounds like I'm doing something with translations for the app. `updateNotificationLocale` may be more appropriate.
2. Do we really need a separate handler function? The only thing this does is call `setNotificationLang` which could be done inline within `onChange`. It isn't removing any duplication either.